rpi_backlight/__init__.py

```diff
@@ -39,9 +60,14 @@ def __init__(
                 )
             backlight_sysfs_path = _EMULATOR_SYSFS_TMP_FILE_PATH.read_text()
         self._backlight_sysfs_path = Path(backlight_sysfs_path)
-        self._max_brightness = self._get_value("max_brightness")  # 255
+        self._board_type = board_type
         self._fade_duration = 0.0  # in seconds
 
+        if self._board_type == BoardType.TINKER_BOARD:
+            self._max_brightness = 255
+        else:
+            self._max_brightness = self._get_value("max_brightness",)  # 255
```

Comment:
Just to be in sync with the other conditions checking `board_type`:

```suggestion
        if self._board_type == BoardType.RASPBERRY_PI:
            self._max_brightness = self._get_value("max_brightness")  # 255
        elif self._board_type == BoardType.TINKER_BOARD:
            self._max_brightness = 255
```

makes it easier to add more board types in the future too (very unlikely though)

also no need to add the comma in `self._get_value("max_brightness",)`.